### PR TITLE
[CFM-481] notify group admins that membership was approved

### PR DESCRIPTION
**Pull request by anvmn**
## This PR is dependent on #1059 

https://issuetracker.amplexor.com/jira/browse/CFM-481

### CODE REVIEW

**Comment by anvmn** [comment]
selection_004

**Comment by ordavidil** [comment]
re-roll

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc [hunk 1]
```diff
@@ -54,11 +54,14 @@ function c4m_user_notifications_moderated_group_notify_admins_membership_accepte
...
  */
 function _c4m_user_notifications_moderated_group_notify_admins_membership_accepted(Entity $membership, $admin_id) {
+  global $user;
   if (!($message = message_create('c4m_notify_admins_membership_approved', array('uid' => $admin_id)))) {
     // If message was not created properly, return early.
     return;
   }
   c4m_message_load_tokens__c4m_notify_admins_membership_accepted($message);
+  $message->arguments['@event_user_full_name'] = c4m_user_name($user);
+  $message->arguments['!event_user_url'] = url('user/' . $user->uid, array('absolute' => TRUE));
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: @anvmn 
should we need to take care of the __PURL__ here on the link?

**Comment by anvmn** [comment]
## Replaced by #1189

**Pull request closed without merging**